tools/accuracy_checker/tests/common.py

```diff
@@ -96,6 +104,16 @@ def make_segmentation_representation(mask, ground_truth=False):
     return [SegmentationPrediction('identifier', mask)]
 
 
+def make_instance_segmentation_representation(mask, labels, ground_truth=False):
```

Comment:
looks like it require specific import, better to move closer to test where it will be used if you do not want handle case when maskUtils is None